### ISSUE TITLE
Spurious warning message when connecting to host

### ISSUE THREAD

**Issue by hholst80** [fabric/fabric#1569]
I see a warning from the Crypto package when I run fabric. It does not seem to be of any problem to execute the command anyway but it can be confusing for our users:

```
$ fab --ssh-config-path=/tmp/ssh_config -H igor apt:jq
[igor] Executing task 'apt'
[igor] run: echo sudo apt-get install -y --no-install-recommends jq
/usr/lib/python2.7/dist-packages/Crypto/Cipher/blockalgo.py:141: FutureWarning: CTR mode needs counter parameter, not IV
  self._cipher = factory.new(key, *args, **kwargs)
[igor] out: sudo apt-get install -y --no-install-recommends jq
[igor] out: 


Done.
Disconnecting from labs@igor.frostbite.com... done.
```

The warning can be disabled as a workaround:

```
import warnings
warnings.simplefilter(action="ignore", category=FutureWarning)
```

http://stackoverflow.com/questions/15777951/how-to-suppress-pandas-future-warning

**Comment by ploxiln** [comment]
fixed in https://github.com/paramiko/paramiko/pull/889 - just wait for the next paramiko 1.17.x or 1.18.x release

**Comment by bitprophet** [comment]
Already released there actually, peep 1.17.4/1.18.2 in http://www.paramiko.org/changelog.html :)
